client/components/PieChart.js

```diff
@@ -12,25 +12,34 @@ const dataArc = arc()
   .innerRadius(0)
 
 const labelArc = arc()
-  .outerRadius(radius - 40)
-  .innerRadius(radius - 40)
-
-// const colour = scaleOrdinal([
-//   '#98abc5',
-//   '#8a89a6',
-//   '#7b6888',
-//   '#6b486b',
-//   '#ff8c00'
-// ])
+  .outerRadius(radius - 70)
+  .innerRadius(radius - 70)
 
 const chart = pie()
   .sort(null)
   .value(d => d.quantity)
 
-const width = 300
-const height = 300
+const width = 350
+const height = 350
+
+const PieChart = props => {
+  const rawData = props.rowData
```

Comment:
Just for future reference, the PieChart will be expecting 'rowData' in its props.